src/w3iProxy/pushProviders/internalPushProvider.ts

```diff
@@ -55,6 +55,19 @@ export default class InternalPushProvider implements W3iPushProvider {
     return this.pushClient.reject(params)
   }
 
+  public async subscribe(_params: {
+    metadata: PushClientTypes.Metadata
+    account: string
+    onSign: (message: string) => string
+  }) {
+    if (!this.pushClient) {
+      throw new Error(this.formatClientRelatedError('subscribe'))
+    }
+
+    // Noop until we have a real push client implementation ready.
```

Comment:
Can you prefix this with TODO ?

Reply:
Done 👍 Our ESLint rules here are linting against TODOs so need to use multi line comment